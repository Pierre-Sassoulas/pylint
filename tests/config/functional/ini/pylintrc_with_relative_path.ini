[SPELLING]
spelling-dict=en_US
spelling-private-dict-file=spelling/words.txt
