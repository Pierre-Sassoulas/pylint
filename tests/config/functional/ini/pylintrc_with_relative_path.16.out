************* Module file_to_lint
tests/config/file_to_lint.py:1:0: C0401: Wrong spelling of a word 'GPL' in a comment:
# Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
                     ^^^
Did you mean: ''GP' or 'PL' or 'CPL' or 'GPA''? (wrong-spelling-in-comment)
